workspaces/sandbox/plugins/sandbox/src/components/SignIn/RHSSOSignInPage.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { SignInPageProps } from '@backstage/core-plugin-api';
+import { SignInPage, SignInProviderConfig } from '@backstage/core-components';
+import React from 'react';
+import { keycloakApiRef } from '../../api';
```

Comment:
```suggestion
import React from 'react';
import { SignInPageProps } from '@backstage/core-plugin-api';
import { SignInPage, SignInProviderConfig } from '@backstage/core-components';
import { keycloakApiRef } from '../../api';
```

Reply:
Nit: Rearranging the imports to keep React stuffs on top and local stuffs at the end.